modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/CKEditorCreoleConfigContributor.java

```diff
@@ -85,7 +85,7 @@ public void populateConfigJSONObject(
 		StringBundler sb = new StringBundler(4);
 
 		sb.append("bidi,codemirror,colorbutton,colordialog,div,");
-		sb.append("elementspath,font,forms,indentblock,justify,keystrokes");
+		sb.append("elementspath,font,forms,indentblock,justify,keystrokes,");
```

Comment:
nice catch! 😱

Reply:
With the help of @ambrinchaudhary, it has been very easy ❤️